.travis.yml

```diff
@@ -10,3 +10,8 @@ before_script:
   - git config --global user.name "Travis CI"
 after_script:
   - php composer/bin/test-reporter
+
+addons:
+  code_climate:
+    repo_token:
+      secure: HELLO
```

Comment:
Of course, this needs to be encrypted using https://github.com/travis-ci/travis.rb#encrypt;

```
$ travis encrypt HELLO
```

And, of course, instead of `HELLO` you should use the actual token.